pkg/steps/artifacts.go

```diff
@@ -693,6 +694,35 @@ func gatherContainerLogsOutput(podClient PodClient, artifactDir, namespace, podN
 	return kerrors.NewAggregate(validationErrors)
 }
 
+// for gathering successful build logs to the artifacts, there is no way to augment the pod spec
+// created by the build controller to add the artifacts container; this method cherry picks elements
+// from downloadArtifacts and gatherContainerLogsOutput and munges them in conjunction with the build
+// api logging capabilities; also, without needing to inject an artifacts container, some of the complexities
+// around download/copy from the artifacts container's volume mount and multiple pods are avoided.
+func gatherSuccessfulBuildLog(buildClient BuildClient, artifactDir, namespace, buildName string) error {
+	// adding a subdir to the artifactDir path similar to downloadArtifacts adding the container-logs subdir
+	dir := filepath.Join(artifactDir, "openshift-docker-build-strategy-logs")
```

Comment:
I don't like this name.  I'd prefer we shorten this to `build-logs`.

Reply:
https://github.com/openshift/ci-operator/pull/303